components/tag-list/README.md

```diff
@@ -70,9 +70,17 @@ The `d2l-tag-list-item` provides the appropriate `listitem` semantics and stylin
     e.target.parentNode.removeChild(e.target);
     console.log(`d2l-tag-list-item-clear event dispatched. Value: ${e.detail.value}`);
   });
+
+  document.addEventListener('d2l-tag-list-clear', (e) => {
+    const items = e.target.querySelectorAll('[role="listitem"]');
+    items.forEach((item) => {
+      item.parentNode.removeChild(item);
+    });
+    console.log('d2l-tag-list-clear event dispatched');
+  });
```

Comment:
The "Clear All" button won't show up in this demo since there is only a single item and it only shows up when there are 4+ items.

I think you may have meant to add an event listener for `d2l-tag-list-item-clear` here instead

Reply:
There was already an event listener for `d2l-tag-list-item-clear` but it was missing the `clearable` attribute. I added the clear all one as well just for completeness.

Reply:
Oops, this was added previously higher up in the code. You can disregard.

Reply:
Oops missed that!